pkgs/development/libraries/fontconfig/make-fonts-conf.nix

```
@@ -1,5 +1,10 @@
-{ runCommand, stdenv, lib, libxslt, fontconfig, dejavu_fonts, fontDirectories
-, impureFontDirectories ? [
+{ runCommand, stdenv, lib, libxslt, fontconfig, dejavu_fonts }:
+
+{
+  # an array of fonts, e.g. `[ pkgs.dejavu_fonts.minimal ]`
+
+  fontDirectories
```

Ah, I didn't notice this: to make the new `makeFontsConf` 100% backward compatible, this function should accept `fontconfig` as optional argument.
```suggestion
  fontconfig ? fontconfig
,  fontDirectories
```

---

(You could add a relevant warning and a deprecation notice, but... it doesn't seem worth the effort.)